e2e-test/cypress/integration/responsive.spec.js

```diff
@@ -4,6 +4,12 @@ describe('Responsive Image', () => {
     cy.visit('/');
     cy.get('#responsiveBtn').click();          // Click on button
   });
+  it('Responsive Image Override', () => {
```

Comment:
Can you explain what you mean by "responsive override"?

Reply:
Added comments